database/Cargo.toml

```diff
@@ -22,9 +22,9 @@ tracing = "0.1.13"
 tokio = { version = "1.1", features = ["sync", "time"] }
 
 actix-diesel = { git = "https://github.com/frol/actix-diesel", rev = "3a001986c89dfabfc3c448d8bae28525101b4992" }
-near-indexer-primitives = "0.14"
-near-primitives = "0.14"
-near-crypto = "0.14"
+near-indexer-primitives = "0.16.0"
```

Comment:
FYI we expect `0.16.1` to be released with re-exporting of the entire `near-primitives` to be done before `1.32.0` nearcore release.

Reply:
It seems they have forgotten about it, so we're flying without 0.16.1